deployments/outputs.json

```diff
@@ -36,5 +36,45 @@
       "last_deployment_stage": 5,
       "network_id": 42
     }
+  },
+  "50-development": {
```

Comment:
unstage these in source tree before committing

Reply:
if you run `yarn delete-developments` it should wipe it